<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RouteSettingsWidget</class>
 <widget class="QWidget" name="RouteSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Routing Rules</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Rule List</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="rulesListbox"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="addRuleBtn">
             <property name="icon">
              <iconset>
               <normaloff>:/assets/icons/ui_dark/add.svg</normaloff>:/assets/icons/ui_dark/add.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="removeRuleBtn">
             <property name="icon">
              <iconset>
               <normaloff>:/assets/icons/ui_dark/minus.svg</normaloff>:/assets/icons/ui_dark/minus.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="moveRuleUpBtn">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/assets/icons/ui_dark/arrow-up.svg</normaloff>:/assets/icons/ui_dark/arrow-up.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="moveRuleDownBtn">
             <property name="icon">
              <iconset>
               <normaloff>:/assets/icons/ui_dark/arrow-down.svg</normaloff>:/assets/icons/ui_dark/arrow-down.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="dnsSettingsGB">
         <property name="title">
          <string>Rule Settings</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_51">
              <property name="text">
               <string>Disabled</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="disabledCB"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Name</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="nameTxt"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_52">
              <property name="text">
               <string>Domain</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="domainTxt">
              <property name="placeholderText">
               <string>split with "|"</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_53">
              <property name="text">
               <string>IP</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="ipTxt">
              <property name="placeholderText">
               <string>split with "|"</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_54">
              <property name="text">
               <string>Port</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="portTxt"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_55">
              <property name="text">
               <string>Source Port</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="srcPortTxt"/>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_56">
              <property name="text">
               <string>Network</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QComboBox" name="networkCombo">
              <item>
               <property name="text">
                <string notr="true">tcp,udp</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string notr="true">tcp</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string notr="true">udp</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="label_57">
              <property name="text">
               <string>Source IP</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QLineEdit" name="sourceIpTxt">
              <property name="placeholderText">
               <string>split with "|"</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QLabel" name="label_58">
              <property name="text">
               <string>Protocol</string>
              </property>
             </widget>
            </item>
            <item row="8" column="1">
             <widget class="QLineEdit" name="protocolTxt">
              <property name="placeholderText">
               <string>split with "|"</string>
              </property>
             </widget>
            </item>
            <item row="9" column="0">
             <widget class="QLabel" name="label_59">
              <property name="text">
               <string>Outbound</string>
              </property>
             </widget>
            </item>
            <item row="9" column="1">
             <widget class="QComboBox" name="outboundCombo">
              <item>
               <property name="text">
                <string notr="true">Proxy</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string notr="true">Direct</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string notr="true">Block</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
